client/src/components/ViewMyProfile.js

```diff
@@ -1,136 +1,173 @@
-	const {profile} = useContext(ProfileContext)
-
-	const handleClick = ()=>{
-		history.push('/editprofile');
-	};
-
-	return (
-		<Container>
-			<Circle/>
-			<SubCon>
-				<Name>{profile.first_name} {profile.last_name}</Name>
-				{isAuthenticated&&<StyledButton name='Edit profile' handleClick={handleClick} className='lg'/>}
-				<Description>
-				Dolorem ipsum quia dolor sit amet, excepturi sint  quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit,.
-				</Description>
-				<SubHeads>{profile.first_name}’s skills</SubHeads>
-					<SkillsContainer>
-						{profile.skills&&profile.skills.map((skill)=>{
-							return (
-								<SkillBox>{skill}</SkillBox>	
-							)
-						})}
-					</SkillsContainer>
-					<SubHeads>Find {profile.first_name}</SubHeads>
-				<SocialCon>
-					<SocialSubCon>
-						<SocialIcon/>
-						<SocialText>LinkedIn</SocialText>
-					</SocialSubCon>
-					<SocialSubCon>
-						<SocialIcon/>
-						<SocialText>GitHub</SocialText>
-					</SocialSubCon>
-					<SocialSubCon>
-						<SocialIcon/>
-						<SocialText>CV</SocialText>
-					</SocialSubCon>
-				</SocialCon>
-			</SubCon>
-		</Container>
-	);
+import React, { useContext } from "react";
+import { useHistory } from "react-router-dom";
+import { AuthContext } from "../context/AuthContext";
+import { ProfileContext } from "../context/ProfileContext";
+import styled from "styled-components";
+import GitHubIcon from "./GitHubIconIcon.js";
+import LinkedinIcon from "./LinkedinIcon";
+import CvIcon from "./CvIcon";
+import EmailIcon from "./EmailIcon";
+import RichEditorReader from '../constant/RichEditorReader'
+
+const ViewMyProfile = ({ profile }) => {
+
+  let history = useHistory();
+  const { isAuthenticated } = useContext(AuthContext);
+
+  const handleClick = () => {
+    history.push("/profile/edit");
+  };
+  console.log('avatar', profile.avatar_url)
+  return (
+    <Container>
+      <Img src={profile.avatar_url}/>
+      <SubCon>
+        <Name>
+        {profile.first_name} {profile.surname}
+        {profile.is_hired && <HiredLabel>Hired</HiredLabel>}
+        </Name>
+        <Description>{profile.location}</Description>
+        <IconContainer>
+          <GitHubIcon gitHubLink={profile.github_link}></GitHubIcon>
+          <LinkedinIcon linkedinLink={profile.linkedin_link}></LinkedinIcon>
+          <CvIcon></CvIcon>
+          <EmailIcon profile={profile}></EmailIcon>
+        </IconContainer>
+        <DescHead>About Me</DescHead>
+        <Description>{profile.about_me}</Description>
+        <SubHeads> {profile.first_name}'s Interests</SubHeads>
+        <Description>{profile.interest}</Description>
+        <SubHeads>{profile.first_name}’s skills</SubHeads>
+        <SkillsContainer>
+          {profile.skills && profile.skills.map((skill) => {
+            return <SkillBox>{skill}</SkillBox>;
+          })}
```

Comment:
There's nothing wrong with this but you could save two lines:
```suggestion
          {profile.skills && profile.skills.map((skill) => <SkillBox>{skill}</SkillBox>)}
```